src/schemas/advertising.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$id": "https://meta.comcast.com/firebolt/advertising",
```

Comment:
How come this got in here?

Reply:
will check it out

Reply:
I touched a bunch of schemas during this feature and moved them out of `fireboat-schemas` because that's our plan.